reana_workflow_engine_yadage/tasks.py

```diff
@@ -119,3 +121,15 @@ def run_yadage_workflow(workflow_uuid, workflow_workspace,
             message=str(e))
     finally:
         db_session.close()
+        # since we don't know the yadage workflow name upfront we get the
+        # unkonw directory. Fix using yadagge API to retrieve the workflow
+        # name.
+        yadage_workflow_dir = \
+            set(os.listdir(workflow_workspace)).difference(known_dirs).pop()
```

Comment:
@lukasheinrich can we avoid this and get the exact name of the workflow directory from `yadage` API?